src/encoded/schemas/functional_characterization_experiment.json

```diff
@@ -358,7 +358,7 @@
             "length": "long",
             "open_on_load": true
         },
-        "replicates.library.biosample.donor.organism.scientific_name": {
+        "organism.scientific_name": {
```

Comment:
Do we also want to replace with this in the columns as well? I know this is for making the facets compatible, but in the future not sure if this will make a common report more easily compatible as well?

Reply:
updated columns

Reply:
reverting it back, it breaks search and I think we will handle columns in a separate ticket